backend/internal/application/store_constants.go

```diff
@@ -16,7 +16,7 @@
  * under the License.
  */
 
-package store
+package application
```

Comment:
If we're renaming this file with underscores, shall we rename other files also in the application package?